spec/cipherstash/analysis/text_processor_spec.rb

```diff
@@ -56,6 +56,19 @@
       }.to raise_error(CipherStash::Client::Error::InvalidSchemaError, "'tokenLength' is deprecated. Use 'minLength' and 'maxLength' for the ngram filter.")
     end
 
+    it "splits text into ngrams when min and max length are equal" do
+      tokenizer =
+        CipherStash::Analysis::TextProcessor.new({
+          "tokenFilters" => [
+            { "kind" => "downcase" },
+            { "kind" => "ngram", "minLength" => 3, "maxLength" => 3 }
+          ],
+         "tokenizer" => { "kind" => "standard" }
+        })
+      result = tokenizer.perform("Example")
+      expect(result).to eq(["exa", "xam", "amp", "mpl", "ple", "example"])
```

Comment:
Does the full string always show up as a token even when it's greater than the max length? I noticed that `"example"` is included in the tokens here.

Reply:
yes, Dan suggested that so we can use the bloom filter for an exact match.